src/styles/index.scss

```diff
@@ -2,2 +1,152 @@
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  &:hover {
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
```

Comment:
The class `&__image` has hard-coded values for dimensions and margins. If these values are shared with other elements or are part of a design system, consider using variables to define them.